rqt_py_common/src/rqt_py_common/item_delegates.py

```diff
@@ -66,7 +66,7 @@ class DelegateUtil(object):
     def _get_hierarchy_level(model_index):
         hierarchyLevel = 1
         seek_root = model_index
-        while(seek_root.parent() != QModelIndex()):
+        while (seek_root.parent() != QModelIndex()):
```

Comment:
I'll point out that the parentheses aren't required at all here:

```suggestion
        while seek_root.parent() != QModelIndex():
```